source/docs/advanced/theme/wysiwyg-platform.md

```diff
@@ -29,6 +30,18 @@ The goal of this Wysiwyg type is to remain as simple as possible. It is not mean
 
 The MagentoWyswiyg's goal is to support all the default features in Magento 1 & 2. If you notice that some features are missing, please contact us and we'll look into implementing it.
 
+<blockquote class="info">
+If you are using the Prismic module, then you need to override `theme/modules/WysiwygV2/getWysiwygComponent.js` and map both of the Wysiwyg customizations
```

Comment:
**nitpick:** I'm afraid it adds some unnecessary details for readers wanted to know details on the `MagentoWysiwyg`

**suggestion:** remove this part, and maybe add it to [the Prismic module Installation page](https://deploy-preview-394--heuristic-almeida-1a1f35.netlify.app/docs/prismic/installation.html) as an Optional section (that would redirect the reader to https://deploy-preview-394--heuristic-almeida-1a1f35.netlify.app/docs/prismic/embed-fields.html#Wysiwyg-Embed-Fields too, as per my other comment)

**thought:** later, I think the `MagentoWysiwyg` should also be relocated under a Magento specific section. But that's unrelated to this MR, and could be tackled when we rework the documentation for making it less _Magentish_ :)